### DISCUSSION TITLE
(Lesson 2)rt_mod = tvm.build(sch.mod,target="llvm")报错

### DISCUSSION THREAD

**Discussion by tfruan2000** [mlc-ai/mlc-zh#110] [Q&A] [answered]
视频(https://www.bilibili.com/video/BV1kY411N7JA)   35:50处，
代码 rt_mod = tvm.build(sch.mod,target="llvm") 报错如下:
**Check failed: (global_symbol) is false: MakePackedAPI: Expect PrimFunc to have the global_symbol attribute**

sch.mod.script()输出如下：
@tvm.script.ir_module
class Module:
    @T.prim_func
    def main(A: T.Buffer[128, "float32"], B: T.Buffer[128, "float32"], C: T.Buffer[128, "float32"]) -> None:
        # function attr dict
        T.func_attr({"gobal_symbol": "main", "tir.noalias": True})
        # body
        # with T.block("root")
        for i_0 in T.parallel(8):
            for i_2, i_1 in T.grid(4, 4):
                with T.block("C"):
                    vi = T.axis.spatial(128, i_0 * 16 + i_1 * 4 + i_2)
                    T.reads(A[vi], B[vi])
                    T.writes(C[vi])
                    C[vi] = A[vi] + B[vi]

**Comment by Hzfengsy** [comment, marked as answer]
typo “gobal_symbol”

**Reply by tfruan2000** [reply]
谢谢！我看见了，我不小心打错了qwq
